packages/devops/scripts/lambda/startTaggedInstances.py

```diff
@@ -0,0 +1,32 @@
+import boto3
+import asyncio
+import functools
+import time
+from utilities import *
+
+ec2 = boto3.resource('ec2')
+ec = boto3.client('ec2')
+iam = boto3.client('iam')
+route53 = boto3.client('route53')
+loop = asyncio.get_event_loop()
+
+async def wait_for_instance(instance_id, to_be):
```

Comment:
Import from utilities instead?

Reply:
Oops, good catch - that's the whole reason I moved that function to utilities 🤦